pom.xml

```diff
@@ -2007,6 +2009,22 @@
                 <artifactId>javassist</artifactId>
                 <version>3.29.2-GA</version>
             </dependency>
+            <dependency>
+                <groupId>org.junit.platform</groupId>
+                <artifactId>junit-platform-launcher</artifactId>
+                <version>${junit-platform-launcher.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-engine</artifactId>
+                <version>${junit-jupiter.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.junit.vintage</groupId>
+                <artifactId>junit-vintage-engine</artifactId>
+                <version>${junit-jupiter.version}</version>
+            </dependency>
+            <!-- Override the JUnit 4 API version provided by junit-vintage-engine -->
```

Comment:
What do you mean? Why can't we use `junit-vintage-engine` only?

Reply:
We better specify the version we want. Otherwise  cache-tests dependency is bringing junit:4.11 , kubernetes-server-mock dependency is bringing junit:4.12 etc.